### ISSUE TITLE
Integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3

### ISSUE THREAD

**Issue by kmichels** [briis/hass-weatherflow#56]
Restarting my HA installation today threw this error and said to report it:

```
Logger: homeassistant.helpers.frame
Source: helpers/frame.py:77 
First occurred: 12:26:14 PM (1 occurrences) 
Last logged: 12:26:14 PM

Detected integration that called async_setup_platforms instead of awaiting async_forward_entry_setups; this will fail in version 2023.3. Please report issue to the custom integration author for weatherflow using this method at custom_components/weatherflow/__init__.py, line 167: hass.config_entries.async_setup_platforms(entry, WEATHERFLOW_PLATFORMS)
```

**Comment by bigfootea1** [comment]
Mine too!

**Comment by briis** [comment]
Closed with release 1.0.11
